client/bare_metal_client.go

```diff
@@ -144,7 +144,6 @@ type BareMetalClient interface {
 	ListSecurityLists(compartmentID, vcnID string, opts *baremetal.ListOptions) (res *baremetal.ListSecurityLists, e error)
 	ListShapes(compartmentID string, opts *baremetal.ListShapesOptions) (shapes *baremetal.ListShapes, e error)
 	ListSubnets(compartmentID, vcnID string, opts *baremetal.ListOptions) (subnets *baremetal.ListSubnets, e error)
-	ListSupportedOperations() (*baremetal.ListSupportedOperations, error)
```

Comment:
This was a provisional whitelisting feature now subsumed by identity capabilities api, good to remove.